Formula/libzt.rb

```diff
@@ -0,0 +1,32 @@
+class Libzt < Formula
+  desc "Encrypted P2P networking library for applications (GPLv3)"
+  homepage "https://www.zerotier.com"
+
+  url "https://github.com/zerotier/libzt.git",
+    :tag      => "1.3.1-hb0",
+    :revision => "26ba6add126dd425c73627706753044e10b606af"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "git", "-C", "ext/lwip", "apply", "../lwip.patch"
+    system "git", "-C", "ext/lwip-contrib", "apply", "../lwip-contrib.patch"
```

Comment:
I do not thing non-default patches should be applied by default in the Homebrew formula.

Reply:
Understood. I've forked the projects in question, applied the patches there, and updated my project's submodules.